apis/cosmos-source.js

```diff
@@ -569,7 +579,7 @@ export default class CosmosAPI {
     const rewards = (result.rewards || []).filter(
       ({ reward }) => reward && reward.length > 0
     )
-    return this.reducers.rewardReducer(rewards, this.validators)
+    return await this.reducers.rewardReducer(rewards, this.validators)
```

Comment:
is await needed?

Reply:
yes, it is an `async` function, this reducer